### PR TITLE
W-13830978 pdk overiew gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by JaatadiaMulesoft — APPROVED** [review]

File: gateway/1.5/modules/ROOT/pages/policies-pdk-overview.adoc [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
+
+Policies enforce regulations to help manage security, control traffic, and improve API adoption. Flex Gateway provides many xref:policies-included-directory.adoc[included policies], but they might not cover all your specific business needs.
+
+The Policy Development Kit (PDK) is a software development kit that enables Flex Gateway users to develop custom policies with ease.
+
+PDK provides step by step instructions to guide you through the policy development lifecycle, from implementing your Rust source code to releasing your policy on Exchange. Like included policies, you can apply custom policies to Flex Gateway by using API Manager or YAML configuration files.
```
Comment (JennyHajee) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-overview.adoc]:
```suggestion
PDK provides step-by-step instructions to guide you through the policy development lifecycle, from implementing your Rust source code to releasing your policy on Exchange. Like included policies, you can apply custom policies to Flex Gateway by using API Manager or YAML configuration files.
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-overview.adoc [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+
+PDK also provides template policies to help you get started with policy development.
+
+To begin using PDK, start by:
+
+* xref:policies-pdk-architecture.adoc[Reviewing the architecture]
+* xref:policies-pdk-prerequisites.adoc[]
```
Comment (JennyHajee) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-overview.adoc]:
```suggestion
* xref:policies-pdk-prerequisites.adoc[Reviewing the prerequisites]
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-architecture.adoc [hunk 3]
```diff
@@ -2,4 +2,41 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+The Policy Development Kit (PDK) consists of the:
```
Comment (JennyHajee) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-architecture.adoc:7]: I suggest adding more information about the purpose of this PDK. For example:
The Policy Development Kit (PDK) is a set of templates that enable who to do what, then follow with the content from line 7 and on. 
Overall, I think this is a good overview

File: gateway/1.5/modules/ROOT/pages/policies-pdk-overview.adoc [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+
+Policies enforce regulations to help manage security, control traffic, and improve API adoption. Flex Gateway provides many xref:policies-included-directory.adoc[included policies], but they might not cover all your specific business needs.
+
+The Policy Development Kit (PDK) is a software development kit that enables Flex Gateway users to develop custom policies with ease.
+
+PDK provides step by step instructions to guide you through the policy development lifecycle, from implementing your Rust source code to releasing your policy on Exchange. Like included policies, you can apply custom policies to Flex Gateway by using API Manager or YAML configuration files.
+
+PDK also provides template policies to help you get started with policy development.
+
+To begin using PDK, start by:
```
Comment (JennyHajee) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-overview.adoc]: Maybe add the Before You Begin Using PDK header here

**Review by JennyHajee — APPROVED** [review]

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**